mm2src/coins/utxo.rs

```diff
@@ -1990,10 +1990,16 @@ fn script_sig(message: &H256, key_pair: &KeyPair, fork_id: u32) -> Result<Bytes,
     Ok(sig_script)
 }
 
-pub fn output_script(address: &Address) -> Script {
+pub fn output_script(address: &Address, is_p2pkh: bool) -> Script {
```

Comment:
The `ScriptType` enum will be more clear than the boolean, could you refactor this, please?

Reply:
I thought about adding the `ScriptType` enum as a field to the `Address` struct as we already have this enum here https://github.com/KomodoPlatform/atomicDEX-API/blob/3b333323c0a19c39dd891b799ec6d13a55fbe849/mm2src/mm2_bitcoin/keys/src/address.rs#L21
but this will be a slightly bigger refactor as it will affect more code. what do you think?

Reply:
It's a good idea, but will take too much effort for now :slightly_smiling_face: As discussed in the call, please leave a TODO here, we can do this refactor later.